eng/pipelines/templates/steps/java-publishing.yml

```diff
@@ -0,0 +1,31 @@
+parameters:
+  ArtifactDirectory: not-specified
+  RepositoryUrl: not-specified
+  GroupID: not-specified
+  ArtifactID: not-specified
+  BuildToolsPath: ''
+
+steps:
+  - ${{if eq(parameters.Target, 'Burner')}}:
+    - pwsh: |
+        ${{parameters.BuildToolsPath}}/scripts/Publish-MavenPackages.ps1 -ArtifactDirectory ${{parameters.ArtifactDirectory}} -GroupIDFilter ${{parameters.GroupID}} -ArtifactIDFilter ${{parameters.ArtifactID}} -RepositoryUrl https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-$(Build.BuildID)/maven/v1 -RepositoryUsername nobody -RepositoryPassword $(System.AccessToken) -GPGExecutablePath ${{parameters.BuildToolsPath}}/tools/gpg/bin/gpg.exe -InformationAction Continue
+      workingDirectory: $(Agent.BuildDirectory)
+      displayName: 'Publish to Azure Artifacts'
+
+  - ${{if eq(parameters.Target, 'JavaDevFeed')}}:
+    - pwsh: |
+        ${{parameters.BuildToolsPath}}/scripts/Publish-MavenPackages.ps1 -ArtifactDirectory ${{parameters.ArtifactDirectory}} -GroupIDFilter ${{parameters.GroupID}} -ArtifactIDFilter ${{parameters.ArtifactID}} -RepositoryUrl https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-java/maven/v1 -RepositoryUsername nobody -RepositoryPassword $(System.AccessToken) -GPGExecutablePath ${{parameters.BuildToolsPath}}/tools/gpg/bin/gpg.exe -InformationAction Continue
```

Comment:
To enable better reading without scrolling can you enable link wrapping for these steps.